junifer/data/coordinates/_ants_coordinates_warper.py

```diff
@@ -59,7 +59,8 @@ def warp(
         )
         np.savetxt(
             pretransform_coordinates_path,
-            seeds,
+            # convert to LPS coordinate system
+            np.apply_along_axis(lambda x: x * [-1, -1, 1], 1, seeds),
```

Comment:
can we just do `x[0] *= -1`... It's more easy to read than a lambda function to multiply two axis by -1 and another axis by 1.

Reply:
Not sure I follow, could you please show what L63 should be?